### DISCUSSION TITLE
How to setup a thread network with multiple OTBR?

### DISCUSSION THREAD

**Discussion by olavt** [openthread/openthread#10581] [Q&A]
I'm quite ok with the process of setting up a new thread network using one OpenThread Border Router running on a Raspberry Pi 4.

How do I add another OTBR to the network?

My goal is to extend the network to another part of the house where the Thread devices could route traffic via another OTBR.

**Comment by superwhd** [comment]
Basically you just need to set the second OTBR to use the same active operational dataset as the first OTBR.

On the first OTBR, copy the hex string output of `ot-ctl dataset active -x`.

On the second OTBR,
```shell
$ ot-ctl
> factoryreset                   # To clear the dataset if any
> dataset set active <the hex string you copied>
> ifconfig up
> thread start
```
Then the second OTBR should join the first OTBR's Thread network in a few seconds.

**Reply by olavt** [reply]
Great! How can I join a Raspberry Pi based OTBR to the same Thread network as Apple TV?

**Reply by abtink** [reply]
You need to get/set the credential (operational dataset) from Apple TV (or other devices)

To get or set the credential (operational dataset) from Apple TV, you'll likely need a mobile app. You can refer to the Apple APIs: https://developer.apple.com/documentation/threadnetwork

For Android devices, you can use similar APIs: https://developers.home.google.com/thread
